x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview_empty.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useValues } from 'kea';
+
+import { i18n } from '@kbn/i18n';
+import {
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentBody,
+  EuiTitle,
+  EuiText,
+  EuiButton,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { EngineLogic } from '../engine';
+
+import { DOCS_PREFIX } from '../../routes';
+import {
+  DOCUMENT_CREATION_DESCRIPTION,
+  DOCUMENT_API_INDEXING_TITLE,
+  DOCUMENT_API_INDEXING_DESCRIPTION,
+} from '../document_creation/constants';
+// TODO
+// import { DocumentCreationButtons, CodeExample } from '../document_creation'
+
+export const EmptyEngineOverview: React.FC = () => {
+  const { engineName } = useValues(EngineLogic);
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l">
+            <h1>
+              {i18n.translate('xpack.enterpriseSearch.appSearch.engine.overview.empty.heading', {
+                defaultMessage: 'Engine Setup',
```

Comment:
Per the Kibana writing guidelines, I *think* stuff like this should not be Title Cased. I'm actually unsure of this one.

```suggestion
                defaultMessage: 'Engine setup',
```

Reply:
@chriscressman Do you mind weighing in on this here? Should we be updating various headings and buttons to use sentence case over title case as we migrate to Kibana?

Reply:
Yes, we should consistently use sentence case in all our UIs. This is a company-wide style rule. The Kibana migration is a great opportunity to align. Thanks for checking!

Reply:
Also @JasonStoltz - should `Engine overview` also be sentence cased vs title cased in that case? 🤔

Reply:
Yes, I think ideally we would change all heading and button text within these screens to use sentence case.

@gchaps is the expert on Kibana (and Elastic) UI text. Can you confirm Gail?

Reply:
Thanks Chris! I think my confusion is what's considered a "Feature Title" vs not. "Overview" is technically the title of the page and is capitalized in the navigation, does that make it a feature that should remain capitalized?

Reply:
Sorry for opening this can of worms 😬